src/sentry/api/endpoints/accept_project_transfer.py

```diff
@@ -58,11 +58,8 @@ def get(self, request: Request) -> Response:
         except InvalidPayload as e:
             return Response({"detail": str(e)}, status=400)
 
-        organizations = Organization.objects.filter(
-            status=OrganizationStatus.ACTIVE,
-            id__in=OrganizationMember.objects.filter(
-                user=request.user, role=roles.get_top_dog().id
-            ).values_list("organization_id", flat=True),
+        organizations = Organization.objects.get_member_top_dog_organizations(
```

Comment:
"get member top dog organizations" 🤔 

I still don't know what this is supposed to return to me....

Is this 
"get me all of the organizations for which i am the top dog"???

Reply:
changing it `get_organizations_where_user_is_owner`